spec/lib/services/pollers-api-service-spec.js

```diff
@@ -0,0 +1,258 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+"use strict";
+
+describe("Http.Services.Api.Pollers", function () {
+    var pollerService;
+    var waterline;
+    var taskProtocol;
+
+    before("Http.Services.Api.Pollers before", function() {
+        helper.setupInjector([
+            helper.require("/lib/services/pollers-api-service.js")
+        ]);
+
+        waterline = helper.injector.get('Services.Waterline');
+        waterline.workitems = {
+            needByIdentifier: sinon.stub().resolves(),
+            find: sinon.stub().resolves([]),
+            create: sinon.stub().resolves(),
+            updateByIdentifier: sinon.stub().resolves(),
+            destroyByIdentifier: sinon.stub().resolves()
+        };
+
+	    taskProtocol = helper.injector.get("Protocol.Task");
+        taskProtocol.requestPollerCache = sinon.stub();
+
+        pollerService = helper.injector.get("Http.Services.Api.Pollers");
+
+    });
+
+    describe("getPollers", function() {
+        it('should expose the appropriate methods', function() {
+            pollerService.should.have.property('getPollers')
+                .that.is.a('function').with.length(1);
```

Comment:
Thanks, it's my fault, I didn't realized the function has a property `length`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/length